src/dodal/devices/i24/dual_backlight.py

```diff
@@ -17,27 +39,26 @@ class DualBacklight(Device):
 
     To set the position for LED1:
         b = DualBacklight(name="backlight)
-        b.pos1.pos_level.set("OAV2")
+        b.bl_position.pos_level.set("OAV2")
```

Comment:
Could: You can just do `b.bl_position.set("OAV2")` because you have overridden the set